walkthroughs/publishing-walkthroughs/walkthrough.adoc

```diff
@@ -125,6 +125,17 @@ If the git repository is updated with new content, the web app is not automatica
 
 ====
 
+[TIP]
+====
+You can also configure the set of {walkthrough} repositories published to a cluster using the command line:
+
+. Log into the cluser using the `oc` command.
+. Enter the following command, where <locations> specifies the repositories and branches as above:
+----
+oc patch webapp tutorial-web-app-operator -n webapp --type=merge -p '{ "spec": { "template": { "parameters": { "WALKTHROUGH_LOCATIONS": "<locations>" }}}}'
```

Comment:
chances are the user (pds or osd) won't know what the installer has done. 
TBH, I wonder if we should maybe move this info to 
* user-documentation, where we know the installation environment
* Paul M's getting started for PDS where we know the installation env
WDYT?

Reply:
That could make sense, unless we're ok with being vague with the webapp namespace in this document instead of giving the actual namespace identifier. Like:

```
oc patch webapp tutorial-web-app-operator -n <webapp-namespace> --type=merge -p '{ "spec": { "template": { "parameters": { "WALKTHROUGH_LOCATIONS": "<locations>" }}}}'
```

If we move the docs to a place where the user (or we) know what has been done with the `ns_prefix` then the documentation may be clearer.